pydra/engine/core.py

```diff
@@ -260,7 +277,7 @@ def __call__(self, submitter=None, plugin=None, **kwargs):
         return res
 
     def _run(self, **kwargs):
-        self.inputs = dc.replace(self.inputs, **kwargs)
+        # self.inputs = dc.replace(self.inputs, **kwargs) don't need it?
```

Comment:
this was there to ensure that inputs could be set via the run command (when it was not a private function) or as a function call. as long as this is the case within `__call__` it would be ok to remove this. i.e. a user should be able to set inputs during the call to a task (much like a python function). 

if `**kwargs` is not being used, it should be removed.

Reply:
oh, i see, we have `kwargs` in the `__call__` that is passing to `_run`, but had no tests for it, so I couldn't find who is using this. will keep it and add a test

Reply:
added [a test](https://github.com/nipype/pydra/pull/130/files#diff-871957728874435d337bd031b6bcb53cR216)